niworkflows/viz/plots.py

```diff
@@ -29,21 +29,23 @@
 import matplotlib.pyplot as plt
 from matplotlib import gridspec as mgs
 import matplotlib.cm as cm
+import matplotlib.patches as mpatches
```

Comment:
I assume this used to generate the legend - that should go on a different PR.

Reply:
Indeed this module is needed for the legend

Reply:
```suggestion
```